src/main/java/picard/sam/ValidateSamFile.java

```diff
@@ -133,75 +133,117 @@ public static void main(final String[] args) {
         System.exit(new ValidateSamFile().instanceMain(args));
     }
 
+    /**
+     * Return tyeps for doWork()
+     */
+    enum ReturnTypes {
+        FAILED(-1),     // failed to complete execution
+        SUCCESSFUL(0),  // ran successfully
+        WARNINGS(1),    // warnings but no errors
+        ERRORS(2);      // errors and possibly warnings
```

Comment:
I do not have the need, but it does not add too much complexity and it may be useful in some cases to know if warnings and errors should be fixed, or just errors.